docs/recipes/virtualenv.rst

```diff
@@ -0,0 +1,65 @@
+Setting Up a Python Virtualenv for SCons
+----------------------------------------
+
+It is often useful to set up a virtualenv when working with a project
+that uses SCons to build.  A virtualenv is a way to create an
+isolated execution environment - you can install whatever you need
+there, and change versions as needed, without affecting anything
+else on your system that uses Python.  Here is an example session
+for setting one up:
+
+.. code:: shell
+
+   $ cd Work
+   $ python -m venv myvenv
+   $ cd myvenv
+   $ source bin/activate
+   (myvenv) $ python -m pip list --outdated
```

Comment:
Only time I use `python -m pip` is on windows to upgrade pip itself

I'd just use pip here.

Reply:
I can change that; if the virtualenv is active, pip indeed will be "the correct one", I just usually use this form when showing people stuff in order to cultivate good habits.  People get itno a *lot* of trouble just running pip, and then "why won't Python find my module".